### PR TITLE
Make corrections to basetypes, add cql-execution as part of the exports

### PR DESCRIPTION
**Pull request by ljades**
These last changes to cqm-models will allow the JS-ECQM-Engine to calculate properly with our patient model.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR:
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass
- [x] If there were any JavaScript changes, this PR has updated the `dist` directory using `npm run dist`
- [x] If applicable, the library version number in `package.json` and `cqm-models.gemspec` has been updated

**Reviewer 1:**

Name: @holmesie
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name: @okeefm
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 3:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

**Comment by holmesie** [comment]
Could you make both `package.json` and `cql-models.gemspec` say version `0.7.0`, to keep things consistent.

File: templates/patient_extension.js.erb [hunk 1]
```diff
@@ -10,22 +10,6 @@
   extended_data: {},
 }, { id: false });
 
-const PatientSchema = DataElementSchema({
```
Comment (okeefm) [inline templates/patient_extension.js.erb]: Why was this object removed from here? It still seems to be in `Patient.js`

Comment (ljades) [reply templates/patient_extension.js.erb]: This template is an extension on `mongoose_tmeplate.js.erb`, so it gets injected into what's already in that template--including most of this object. If you look in the full version of this extension, you'll see how it starts at halfway down this object, because it's an extension on it.

This was an accidental duplicate created sometime along the way. When updating the template one time I accidentally copy-pasted too much from `Patient.js`. This fixes that issue.

File: app/assets/javascripts/basetypes/Interval.js [hunk 2]
```diff
@@ -7,10 +7,28 @@ function Interval(key, options) {
...
 Interval.prototype.cast = (interval) => {
-  if (typeof interval.low === 'undefined') {
+  if (typeof interval.low === 'undefined' || interval.low === null) {
     throw new Error(`Interval: ${interval} does not have a low value`);
   }
-  return new cql.Interval(interval.low, interval.high, interval.lowClosed, interval.highClosed);
+  const casted = new cql.Interval(interval.low, interval.high, interval.lowClosed, interval.highClosed);
+
+  // Cast Low and High values to Quantities if it is a quantity
+  if (casted.low && casted.low.unit && casted.low.value) {
```
Comment (jbradl11) [inline app/assets/javascripts/basetypes/Interval.js]: Is this assuming that only Quantities will have a `unit` and `value` and that is how this is determining that `casted` is a Quantity? Could you check the type in some way instead?

Comment (ljades) [reply app/assets/javascripts/basetypes/Interval.js]: I don't think so. The JSON objects being casted are just that--JSON with no inherent type except being an object. Having a unit and value is the only way we can really identify them as Quantities.
